apps/full-site-editing/full-site-editing-plugin/block-patterns/class-block-patterns.php

```diff
@@ -41,27 +43,32 @@ public static function get_instance() {
 	}
 
 	/**
-	 * Extends block editor settings to include FSE block patterns.
-	 *
-	 * @param array $settings Default editor settings.
-	 * @return array Filtered editor settings.
+	 * Register FSE block patterns and categories.
 	 */
-	public function register_patterns( $settings ) {
-		// Remove core patterns except 'Two Columns of Text'.
-		$settings['__experimentalBlockPatterns'] = wp_list_filter(
-			$settings['__experimentalBlockPatterns'],
-			array(
-				'title' => 'Two Columns of Text',
-			)
-		);
+	public function register_patterns() {
+		// Register Block Pattern Categories.
+		if ( class_exists( 'WP_Block_Pattern_Categories_Registry' ) ) {
+			register_block_pattern_category( 'blog', array( 'label' => _x( 'Blog', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'call-to-action', array( 'label' => _x( 'Call to Action', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'contact', array( 'label' => _x( 'Contact', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'images', array( 'label' => _x( 'Images', 'Block pattern category', 'gutenberg' ) ) );
+			register_block_pattern_category( 'menu', array( 'label' => _x( 'Menu', 'Block pattern category', 'gutenberg' ) ) );
```

Comment:
Should those textdomains be `full-site-editing` or did you leave them as `gutenberg` on purpose?

Reply:
No, that was a copy-pasta incident. Good spot, thanks!

Reply:
38867e32e66c326e84dfe1b41ee06dd9f776e80e